forge-game/src/main/java/forge/game/card/CardFactoryUtil.java

```diff
@@ -1063,6 +1063,20 @@ public static void addTriggerAbility(final KeywordInterface inst, final Card car
             dethroneTrigger.setOverridingAbility(AbilityFactory.getAbility(abString, card));
 
             inst.addTrigger(dethroneTrigger);
+        } else if (keyword.equals("Double Team")) {
+        
+         	final String doubleteamScript = "Mode$ Attacks | ValidCard$ Card.Self+nonToken | TriggerZones$ Battlefield | ImprintSource$ True | TriggerDescription$(" + inst.getReminderText() + ")";
+        	final String makeString = "DB$ MakeCard | DefinedName$ Self | Zone$ Hand | RememberMade$ True ";
```

Comment:
can you check the indent with these lines? i think 2 of them aren't correct indented